packages/web-components/src/radio-group/README.md

```diff
@@ -42,6 +42,7 @@ Used anywhere an author might group a list of radio options.
 | `name`        | public  | `string`                 |              | The name of the radio group. Setting this value will set the name value for all child radio elements. |
 | `value`       | public  | `string`                 |              | The value of the checked radio.                                                                       |
 | `orientation` | public  | `horizontal \| vertical` | `horizontal` | The orientation of the group                                                                          |
+| `stacked`     | public  | `boolean`                | `false`      | places each radio item in a row, with labels below the radio indicator.                               |
```

Comment:
From the readme it is not clear what renders if I do `orientation="vertical" stacked`.